src/server/routes/transaction/blockchain/getLogs.ts

```diff
@@ -0,0 +1,189 @@
+        "0x0000000000000000000000000000000000000000000000000000000000000000",
+        "0x00000000000000000000000071b6267b5b2b0b64ee058c3d27d58e4e14e7327f",
+      ],
+      data: "0x0000000000000000000000000000000000000000000000000de0b6b3a7640000",
+      blockNumber: "79326434n",
+      transactionHash:
+        "0x568eb49d738f7c02ebb24aa329efcf10883d951b1e13aa000b0e073d54a0246e",
+      transactionIndex: 1,
+      blockHash:
+        "0xaffbcf3232a76152206de5f6999c549404efc76060a34f8826b90c95993464c3",
+      logIndex: 0,
+      removed: false,
+    },
+  ],
+};
+
+export async function getTxLogs(fastify: FastifyInstance) {
+  fastify.route<{
+    Params: Static<typeof requestSchema>;
+    Reply: Static<typeof responseBodySchema>;
+  }>({
+    method: "GET",
+    url: "/transaction/:chain/logs/:txHash",
+    schema: {
+      summary: "Get transaction logs from transaction hash",
+      description: "Get parsed transaction logs from a transaction hash.",
+      tags: ["Transaction"],
+      operationId: "txParseLogs",
+      params: requestSchema,
+      response: {
+        ...standardResponseSchema,
+        [StatusCodes.OK]: responseBodySchema,
+      },
+    },
+    handler: async (request, reply) => {
+      const { chain: requestChain, txHash } = request.params;
+      const chainId = await getChainIdFromChain(requestChain);
+      const chain = await getChain(chainId);
+      const rpcRequest = getRpcClient({ client: thirdwebClient, chain });
+
+      const transactionReceipt = await eth_getTransactionReceipt(rpcRequest, {
+        hash: txHash as `0x${string}`,
+      });
+
+      if (!transactionReceipt) {
+        throw createCustomError(
+          "Transaction not found",
+          StatusCodes.NOT_FOUND,
+          "TRANSACTION_NOT_FOUND",
+        );
+      }
+
+      if (!transactionReceipt.to) {
+        throw createCustomError(
+          "Not a contract transaction",
+          StatusCodes.NOT_FOUND,
+          "CONTRACT_NOT_FOUND",
+        );
+      }
+
+      const contract = getContract({
+        address: transactionReceipt.to,
+        chain,
+        client: thirdwebClient,
+      });
+
+      const abi: AbiEventSignature[] = await resolveContractAbi(contract);
+      const eventSignatures = abi.filter((item) => item.type === "event");
+
+      if (eventSignatures.length === 0) {
+        throw createCustomError(
+          "No events found in contract, or could not resolve contract ABI",
+          StatusCodes.NOT_FOUND,
+          "NO_EVENTS_FOUND",
+        );
+      }
+
+      const preparedEvents = eventSignatures.map((event) =>
+        prepareEvent({
+          signature: event,
+        }),
+      );
+
+      const parsedLogs = parseEventLogs({
+        events: preparedEvents,
+        logs: transactionReceipt.logs,
+      });
+
+      reply.status(StatusCodes.OK).send({ result: parsedLogs });
+    },
+  });
+}
+
+type AbiEventSignature = {
```

Comment:
This might be exported by ABIType already:

https://abitype.dev/api/types#abievent

Does this work?
```typescript
import { AbiEvent } from 'abitype'
```